src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs

```diff
@@ -665,14 +665,11 @@ public void ModifyManifest ([Values (true, false)] bool isRelease)
 	    <Namespace Prefix=""android"" Uri=""http://schemas.android.com/apk/res/android"" />
 	</Namespace>
 </PropertyGroup>
-<ItemGroup>
-	<_Permissions Include=""&lt;uses-permission android:name=&quot;android.permission.READ_CONTACTS&quot; /&gt;"" />
-</ItemGroup>
 <Target Name=""_Foo"">
 	<XmlPoke
 		XmlInputPath=""$(IntermediateOutputPath)android\AndroidManifest.xml""
-		Value=""@(_Permissions)""
-		Query=""/manifest""
+		Value=""12345""
+		Query=""/manifest/@android:versionCode""
```

Comment:
This test was failing.

It actually writes "too much" to the `AndroidManifest.xml` file, removing the `<application>` element, causing the latest version of `bundletool` to error with:

    BT One element <application> was expected, but none were found.

To solve this, let's just set `application:versionCode` in the test instead.

Reply:
Im not sure this is the best solution. The test in question is replicating a situation a customer reported (and is using). So we should try to figure out why this is not working. I'm not sure how the XmPoke code would remove the application element... unless the behaviour of XmlPoke has changed somehow.

Reply:
So the test *should* be appending the permissions to the manifest element and leaving the rest in place. 
If its not doing that then it should be updated to do that somehow.

Having an example of adding `permissions` using `XmlPoke` would be a good thing if we can get it working. Since its a use case which a number of users have been doing.